.github/workflows/python-ci.yml

```diff
@@ -20,8 +20,9 @@ jobs:
       env:
         PIP_INDEX_URL: https://pypi.pacificclimate.org/simple
       run: |
+        sudo apt-get update
         sudo apt purge postgresql-client-13 postgresql-server-dev-13
-        sudo apt-get install libhdf5-serial-dev libnetcdf-dev libspatialite-dev postgresql-9.5-postgis-2.4
+        sudo apt-get install libhdf5-serial-dev libnetcdf-dev libspatialite-dev postgresql-12-postgis-3
```

Comment:
Based on my experience yesterday, I think that you need to `sudo apt purge postgresql-client-13 postgresql-server-dev-all`

Reply:
Thanks :pray: